contrib/mac/framework/julia-config.jl

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env julia
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+const options =
+[
+    "--cflags",
+    "--ldflags",
+    "--ldlibs"
+];
```

Comment:
how common is that? if we built with symlinks by default but you deleted them manually would that work for your use case?

I think a lot of users and package authors aren't dealing with INCLUDEDIR etc correctly yet either. Bikesheddy, but JULIALIBNAME or LIBJULIANAME?